src/Compiler/SyntaxTree/LexHelpers.fs

```diff
@@ -68,33 +75,41 @@ type LongUnicodeLexResult =
     | Invalid
 
 let mkLexargs (conditionalDefines, indentationSyntaxStatus, resourceManager, ifdefStack, diagnosticsLogger, pathMap: PathMap) =
-    { 
-      conditionalDefines = conditionalDefines
-      ifdefStack = ifdefStack
-      indentationSyntaxStatus = indentationSyntaxStatus
-      resourceManager = resourceManager
-      diagnosticsLogger = diagnosticsLogger
-      applyLineDirectives = true
-      stringNest = []
-      pathMap = pathMap
+    {
+        conditionalDefines = conditionalDefines
+        ifdefStack = ifdefStack
+        indentationSyntaxStatus = indentationSyntaxStatus
+        resourceManager = resourceManager
+        diagnosticsLogger = diagnosticsLogger
+        applyLineDirectives = true
+        stringNest = []
+        pathMap = pathMap
     }
 
 /// Register the lexbuf and call the given function
-let reusingLexbufForParsing lexbuf f = 
+let reusingLexbufForParsing lexbuf f =
     use unwindBuildPhase = PushThreadBuildPhaseUntilUnwind BuildPhase.Parse
     LexbufLocalXmlDocStore.ClearXmlDoc lexbuf
     LexbufCommentStore.ClearComments lexbuf
-    
+
     try
-      f () 
+        f ()
     with e ->
-      raise (WrappedError(e, (try lexbuf.LexemeRange with _ -> range0)))
-
-let resetLexbufPos fileName (lexbuf: Lexbuf) = 
-    lexbuf.EndPos <- Position.FirstLine (FileIndex.fileIndexOfFile fileName)
+        raise (
+            WrappedError(
+                e,
+                (try
+                    lexbuf.LexemeRange
+                 with _ ->
+                     range0)
+            )
+        )
```

Comment:
@dsyme @nojaf Could we allow Fantomas to keep such oneliners? Is there a style guide rule for this?

Reply:
This code is badly written, it should use a let binding

I'm OK with try/with always being multiline. There should be fewer of these in the code base anyway